test/rules/no-restricted-globals/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+  "rules": {
+    "no-restricted-globals": [
```

Comment:
Some more test cases to consider...

* `./lib.d.ts`
* `./my-types.d.ts`
* ./`has-global-const-name-declarations.d.ts`
* Functions, variables, interfaces, types, and classes with suspicious names like `name`